core/src/test/scala/unit/kafka/log/LogCleanerIntegrationTest.scala

```diff
@@ -228,6 +231,60 @@ class LogCleanerIntegrationTest(compressionCodec: String) extends AbstractLogCle
     checkLogAfterAppendingDups(log, startSize, appends)
   }
 
+  @Test(timeout = 60000)
```

Comment:
A bit of a long test which I'm not too happy about. Takes about 20-30 seconds locally and it's ran 4 times as the test class is parameterized, but it validates the changes

Reply:
Does this test need to run with all compression codec? It seems running one is enough.

Reply:
No it doesn't. I agree it would be best if we could exclude it.

I've tried to exclude its parameterization but I didn't manage to. 
Following this - https://stackoverflow.com/a/28203229/3218627 - I tried to create two test classes (one parameterized, one not) in an enclosed class but that couldn't compile due to the  `@RunWith(Enclosed.class)` not being able to find the static `parameters` method defined in the `object LogCleanerIntegrationTest` 

```
java.lang.Exception: No public static parameters method on class kafka.log.ComponentTest$LogCleanerIntegrationTest

	at org.junit.runners.Parameterized.getParametersMethod(Parameterized.java:299)
```

I guess another approach would be to create a separate, non-parameterized `LogCleanerIntegrationTest` class. What's your take on this?

Reply:
It seems that we can reduce the timeout now?